src/sentry/services/hybrid_cloud/import_export/impl.py

```diff
@@ -198,15 +199,24 @@ def import_by_model(
                 last_seen_ordinal = min_ordinal - 1
                 for deserialized_object in deserialize("json", json_data, use_natural_keys=False):
                     model_instance = deserialized_object.object
+                    inst_model_name = get_model_name(model_instance)
+
+                    if not isinstance(model_instance, Model):
+                        return RpcImportError(
+                            kind=RpcImportErrorKind.UnexpectedModel,
+                            on=InstanceID(model=str(inst_model_name), ordinal=1),
```

Comment:
nit: imo `ordinal` should be `None` here.

Reply:
should the other one below it also be `None` ?

Reply:
Yeah, it makes sense for them too match. Either way is honestly fine.